src/Service/Config.php

```diff
@@ -112,7 +112,7 @@ protected function getValueFromDb($name)
      */
     protected function castResult($name, $result)
     {
-        if (null !== $result && in_array($name, self::BOOLEAN_NAMES)) {
+        if (null !== $result && in_array($name, self::BOOLEAN_NAMES) && !is_bool($result)) {
```

Comment:
i would switch conditions 2 and 3 around, in order to exit early if a boolean is given
`if (null !== $result && !is_bool($result) && in_array($name, self::BOOLEAN_NAMES))`

Reply:
this was fixed as well, not sure why my note went away.